packages/gitbook/src/lib/api.ts

```diff
@@ -405,18 +424,35 @@ export const getRevision = cache({
             }
         );
 
-        return cacheResponse(response, fetchOptions.metadata ? cacheTtl_7days : cacheTtl_1day);
+        return cacheResponse(response, {
+            ...(fetchOptions.metadata ? cacheTtl_7days : cacheTtl_1day),
+            data: {
+                ...response.data,
+                tags: getResponseCacheTags(response),
+            },
+        });
+    },
+    getKeyArgs: ([spaceId, revisionId, fetchOptions]) => {
+        if (fetchOptions.computed === false) {
```

Comment:
Same thing, where do you use this property?

Reply:
Same